src/commonTest/kotlin/exchange.dydx.abacus/tests/payloads/SkipTrackMock.kt

```diff
@@ -0,0 +1,14 @@
+package exchange.dydx.abacus.tests.payloads
+
+class SkipTrackMock {
```

Comment:
this doesn't need to be a class, can just be an `object`. then you don't need to instantiate this in your test class. can just access like `SkipTrackMock`

Reply:
got it - though FYI i've been following existing patterns so we should probably refactor existing mocks